### PR TITLE
feat: add route to get all products in store

### PR DESCRIPTION
**Pull request by aapetsi**
## Description
Describe change or new feature here.
## Type of change
Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update
## Checklist
Remove any items which are not applicable.
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works .
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by aapetsi** [comment]
[API endpoint for sellers to view products](https://trello.com/c/OWZd9bfR/14-api-endpoint-for-sellers-to-view-products)

File: controllers/product/getProducts.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const Product = require('../../models/product')
+
+function getProducts(req, res) {
+  Product.find()
```
Comment (Rexben001) [inline controllers/product/getProducts.js]: @aapetsi We agreed async/await, can you use async/await for consistency

**Comment by Rexben001** [comment]
@aapetsi Getting all products should include getting all images associated with individual product

File: controllers/product/getProducts.js (hunk 1 above)
Comment (aapetsi) [reply controllers/product/getProducts.js]: @Rexben001 will refactor to async/await

**Pull request merged**